<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="dequeueView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>380</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="enqueuePushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Enqueue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dequeueSizePushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dequeuePushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Dequeue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="frontPushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Front</string>
    </property>
   </widget>
   <widget class="QPushButton" name="isEmptyPushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Is empty</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>380</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="inputPlainTextEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>380</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveValuesPushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Save values</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveValuesPushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Move saved values before selected value</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
